include/dfu/dfu_target.h

```diff
@@ -21,6 +21,7 @@ extern "C" {
 #endif
 
 enum dfu_target_image_type {
+	DFU_TARGET_IMAGE_TYPE_INVALID = 0,
```

Comment:
IMO this should be called `DFU_TARGET_IMAGE_TYPE_ANY` as that is how it is used. That would also improve readability in the code. If we need a specific type for `INVALID` later we should add that in addition to this.

Reply:
updated.